conf.toml

```diff
@@ -5,6 +5,8 @@ Type = "unix"
 
 [InstanceManager]
 Type = "unix"
+HostPort = 1080
```

Comment:
nit: for the future: the prefix Host here, from the context of the Cloud Orchestator Instance Manager is confusing, ideally it would HostOrchestratorServicePort and HostOrchestratorServiceProtocol, unfortunately these names are quite verbose, just something to think about it.